src/Model/AggregatesTrait.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Atk4\Data\Model;
+
+use Atk4\Data\Model;
+
+/**
+ * Provides aggregation methods.
+ */
+trait AggregatesTrait
+{
+    /**
+     * @see Aggregate::withAggregateField.
+     */
+    public function withAggregateField($name, $seed = []): Model
```

Comment:
add phpdoc/strong types, then pass these params, `...func_get_args()` can not be analysed

Reply:
Done